utils/wasm-gen/src/tests.rs

```diff
@@ -176,19 +176,42 @@ fn injecting_addresses_works() {
 fn error_processing_works_for_fallible_syscalls() {
     use gear_backend_common::ActorTerminationReason;
 
+    let _ = env_logger::builder().is_test(true).try_init();
+
+    // We create Unstructured from zeroes here as we just need any
```

Comment:
It was my comment, but anyway)
```suggestion
    // We create Unstructured from zeroes here as we just need any.
```